README.md

```diff
@@ -1,47 +1,52 @@
 # Website
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+The Rancher Docs website is built with [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-### Installation
+## Installation
 
-```
-$ yarn
-```
+Docusaurus requires [Node](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/).
+
+1. [Install](https://nodejs.org/en/download/) Node.
+1. [Install](https://yarnpkg.com/getting-started/install) Yarn.
+1. Go into your local rancher-docs repo folder.
+1. Run `yarn add docusaurus` to finish installation.
+
+> **NOTE:** Although the official Docusaurus 2 documentation states that you should run `npx create-docusaurus@latest my-website classic`, this isn't necessary. The repo already contains all the files Docusaurus needs to build the Rancher Docs website.
```

Comment:
Users should be able to build a local instance of the docs site using what's in the README. They shouldn't need to consult the Docusaurus docs (and specifically the scaffolding/bootstrapping section where that command is found) if their only goal is to run a local instance. Adding the first sentence may just cause confusion.

Reply:
I just pushed a new commit addressing the problems in this section